app/src/main/java/me/saket/dank/data/DankSqliteOpenHelper.kt

```diff
@@ -26,11 +28,16 @@ class DankSqliteOpenHelper(context: Context) : SQLiteOpenHelper(context, DB_NAME
       Timber.d("Resetting cached-message rows")
       // JRAW was bumped to v1.0.
       db.execSQL("DELETE FROM ${CachedMessage.TABLE_NAME}")
+      db.execSQL(AccountManager.QUERY_CREATE_TABLE)
+    }
+
+    if (oldVersion == 2 && newVersion == 3) {
+      db.execSQL(AccountManager.QUERY_CREATE_TABLE)
     }
   }
 
   companion object {
-    private const val DB_VERSION = 2
+    private const val DB_VERSION = 3
     private const val DB_NAME = "Dank"
```

Comment:
I leave it as Dank. We should change it to Dawn or something like that but it will reset all configs for the users out there

Reply:
There is no need for this change. Similarly to package name we would only lose stuff by switching to Dawn so it should be left as is.